src/webviews/src/Jobs.tsx

```diff
@@ -40,41 +41,34 @@ const Jobs: FunctionComponent<JobsProps> = props => {
     .map((j, i) => <Step index={i + 2} step={j as FlatStep} key={i} />)
 
   return (
-    <div className="text-vscode-foreground">
-      <Header
-        title="Where to get data from"
-        description="Flat can fetch data from HTTP endpoints or SQL queries."
-        hasHoverState={false}
-      >
-        {steps}
-
+    <div>
+      <header>
+        <div className="mb-2">
+          <h2 className="my-0 text-[26px] leading-[30px] font-normal">
+            Where to get data from
+          </h2>
+        </div>
+        <p className="text-[13px] my-0">
+          Flat can fetch data from HTTP endpoints or SQL queries.
+        </p>
+      </header>
+      <section className="mt-4">
+        <div className="space-y-6">{steps}</div>
+      </section>
+      <footer className="mt-4">
         <div className="font-bold pb-2">
```

Comment:
I'd also consider using secondary buttons here since there are three primary buttons on one page. Here's how it could look:

<img width="553" alt="CleanShot 2022-01-31 at 16 03 03@2x" src="https://user-images.githubusercontent.com/25163139/151892917-ffad8feb-0776-41af-b11a-36330f105f80.png">

Less pop but perhaps less confusing on an initial scan—depends what you want eyes drawn towards.